src/pages/admin/index.tsx

```diff
@@ -26,14 +28,27 @@ const AdminPage = ({ user: { accessType } }: AdminProps) => {
       </div>
       <br />
       <Typography variant="h2/bold">Store</Typography>
+      <label>
+        <input
+          type="checkbox"
+          defaultChecked={preview === 'member'}
+          onChange={e =>
+            CookieService.setClientCookie(
+              CookieType.PREVIEW,
+              e.currentTarget.checked ? 'member' : 'admin'
+            )
```

Comment:
seeking to learn: should we set the client cookie or the server cookie here? I see in gSSP that we're picking from the server cookie

Reply:
looks like faris answered this on discord

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/d6973708-b715-471f-aba6-065950565f89)